psscriptanalyzer/Invoke-HealthCheck.ps1

```
@@ -0,0 +1,62 @@
+    $output = Invoke-Expression $command
+    return $output
+}
+
+function Get-Statuses {
+    $services = gci -Path Cert:\LocalMachine\My
+    foreach ($svc in $services) {
+        Write-Host "$($svc.Subject) expires $($svc.NotAfter)"
+    }
+}
+
+function Restart-StuckServices {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Restart-StuckServices' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)